app/i18n/en/sub.php

```diff
@@ -183,9 +185,13 @@
 		'method_help' => 'The POST payload has automatic support for <code>application/x-www-form-urlencoded</code> and <code>application/json</code>',
 		'method_postparams' => 'Payload for POST',
 		'moved_category_deleted' => 'When you delete a category, its feeds are automatically classified under <em>%s</em>.',
-		'mute' => 'mute',
+		'mute' => array(
+			'_' => 'mute',
+			'state_is_muted' => 'This feed is muted',
+		),
 		'no_selected' => 'No feed selected.',
 		'number_entries' => '%d articles',
+		'open_feed' => 'Open feed %s',
```

Comment:
I would have preferred `Open feed` instead of `Open feed %s` but no big deal

Reply:
I understand.
Why I chose with the `%s`: It is the tooltip that shows the complete feed name, that could be cropped in the side bar.

Reply:
Alright, makes sense. Would be good to have the same for the user labels